src/main/java/com/the/dark/side/crew/fejsbuk/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package com.the.dark.side.crew.fejsbuk.controller;
+
+import com.the.dark.side.crew.fejsbuk.model.User;
+import com.the.dark.side.crew.fejsbuk.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequiredArgsConstructor
+public class UserController {
+
+    @Autowired
+    private final UserService userService;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<User> getUser(@PathVariable long id) {
+        return new ResponseEntity<>(userService.getUser(id), HttpStatus.OK);
```

Comment:
Let's use statis method: ResponseEntity.ok(userService.getUser()) and if user not found return ResponseEntity.status(NOT_FOUND).build()